token_test.go

```diff
@@ -193,6 +193,21 @@ func TestGetSigningCertificate(t *testing.T) {
 }
 
 func TestTimestamp(t *testing.T) {
+	timestamp := Timestamp{
+		Value:    time.Date(2021, time.September, 17, 14, 9, 10, 0, time.UTC),
+		Accuracy: 5 * time.Second,
+	}
+
+	if !timestamp.Before(time.Date(2021, time.September, 17, 14, 9, 9, 0, time.UTC)) {
+		t.Fatal("timestamp.Before expected true, but got false")
+	}
+
+	if !timestamp.After(time.Date(2021, time.September, 17, 14, 9, 11, 0, time.UTC)) {
+		t.Fatal("timestamp.After expected true, but got false")
+	}
+}
```

Comment:
request more test cases to cover all scenarios.

Reply:
Added more test cases.